test/integration/dynamic_validation_integration_test.cc

```diff
@@ -46,33 +46,37 @@ class TestDynamicValidationNetworkFilterConfigFactory
 
 // Pretty-printing of parameterized test names.
 std::string dynamicValidationTestParamsToString(
-    const ::testing::TestParamInfo<std::tuple<Network::Address::IpVersion, bool>>& params) {
+    const ::testing::TestParamInfo<std::tuple<Network::Address::IpVersion, bool, bool>>& params) {
   return fmt::format(
-      "{}_{}",
+      "{}_{}_{}",
```

Comment:
How long does this test take to run now? Generally wary of uncontrolled combinatorial explosision. If it's not too long it's fine, but prefer to have only a meaningful subset of tests validated with this setting.

Reply:
8.5s (locally on my corp machine). It's in the middle of the pack of //test/integration:all, with the longest ones in the 60s range.